regression/cbmc/CMakeLists.txt

```diff
@@ -4,8 +4,14 @@ else()
   set(gcc_only "")
 endif()
 
+if("{CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+  set(exclude_win_broken_tests -X winbug)
```

Comment:
:pick: :pick: :pick: 
Super nitpicky, but I'm not sure I like the name "winbug" for this. Other than that, no complains here.

Reply:
I agree. I would have chosen something more verbose. Maybe `broken-on-windows`, to be similar to the wording of `broken-smt-backend`. Or maybe `known-windows-bug` to be more similar to original wording.

Reply:
I could change this, but I have a strong preference on doing this in a different PR, not now.

Unless there are strong objections to this.